models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -30,7 +30,9 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
                      outdir = file.path(outdir, runid), 
                      start.year = lubridate::year(stop.time), 
                      end.year = lubridate::year(stop.time),
-                     variables = var.names)
+                     variables = var.names,
+                     stop.time = stop.time,
```

Comment:
If you need to read a specific date why don't read the whole year (without changing the `read.output`) and then subset it here in the `read_restart`  ?